app/experimenter/nimbus-ui/src/components/App/index.tsx

```diff
@@ -3,16 +3,32 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 import React from "react";
-import { Router } from "@reach/router";
+import { Router, Redirect, RouteComponentProps } from "@reach/router";
 
+import { BASE_PATH } from "../../lib/constants";
 import PageHome from "../PageHome";
 import PageNew from "../PageNew";
+import PageEditOverview from "../PageEditOverview";
+import PageEditBranches from "../PageEditBranches";
+import PageRequestReview from "../PageRequestReview";
 
-const App = ({ basepath }: { basepath: string }) => {
+type RootProps = {
+  children: React.ReactNode;
+} & RouteComponentProps;
+
+const Root = (props: RootProps) => <>{props.children}</>;
```

Comment:
We need this since `/edit` is an actual page but doesn't show anything and just redirects. I could stick some "Loading..." text in here and could call it something else but eh, the redirect is super fast.

Reply:
is the enclosing `<>{}</>` necessary?

Reply:
For it to be interpreted as JSX, yeah. If I take it out and change the type of `children` to just `React.ReactChildren`, I get a complaint of "Root cannot be used as a React element." There might be a cleaner way to do this, I'm just not sure what it is after trying different stuff.